### PR TITLE
Activity in ScrollPosition may fails to update drag status.

### PR DESCRIPTION
**Pull request by endless7**
Hi, anyway better to solve this?
details in https://github.com/flutter/flutter/issues/115426

### CODE REVIEW

File: packages/flutter/lib/src/widgets/gesture_detector.dart [hunk 1]
```diff
@@ -1413,20 +1413,6 @@ class RawGestureDetectorState extends State<RawGestureDetector> {
   /// [RawGestureDetector.gestures]. It acts like a temporary replacement for
   /// that value until the next build.
   void replaceGestureRecognizers(Map<Type, GestureRecognizerFactory> gestures) {
-    assert(() {
```
Comment (Piinks) [inline packages/flutter/lib/src/widgets/gesture_detector.dart:1416]: This affects a whole lot more than the single child scroll view, why does this need to be removed for all widgets using gesture detectors?

**Comment by Piinks** [comment]
This does not looks like an appropriate solution for https://github.com/flutter/flutter/issues/115426. I appreciate the contribution though! Can you describe why the issue is happening? Or what the root cause is?

**Comment by endless7** [comment]
Hi - @Piinks, I found the `setPixels` is called when the ballistic animation is over, and the `applyUserOffset` isn't called at this time, which lead to the current ScrollView can still accept gesture events, even the pixels, minScrollExtent and maxScrollExtent are 0.0. Is there any soution better to solve this? Thanks.

**Comment by Piinks** [comment]
> Is there any soution better to solve this?

I am not sure, and unfortunately I can't look into this issue right now. This current proposal appears to be breaking a lot of tests, so I am not sure it is the right way to go.

**Review by Piinks — COMMENTED** [review]
This PR is not currently in a state that it can be reviewed. Can you please look at the currently failing tests?

**Comment by git2321231** [comment]
Any progress? @endless7 @Piinks

**Pull request closed without merging**